tests/scripts/lib/docker.py

```diff
@@ -11,16 +11,25 @@ def __init__(self, cwd='.'):
         self.containers = []
 
     def discover(self):
-        cp = subprocess.run(
-            ['docker', 'compose', 'ps', '--format', 'json'],
-            stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-            cwd=self.cwd,
-        )
-
+        # need to define command required as string to pass in pipe filter '|'
```

Comment:
Ahh, I was trying to make a subprocess of an open subprocess to pass the jq through without the pipe, but was unsuccessful, this would have been a good method to try